rules/threat_intel/threat_intel_rapid7_threat_command.toml

```diff
@@ -0,0 +1,113 @@
+- `threat.indicator.matched.field` - this identifies the vulnerability field that matched the local observation
+- `threat.indicator.matched.type` - this identifies the vulnerability type that matched the local observation
+
+Additional investigation can be done by reviewing the source of the activity and considering the history of the vulnerability that was matched. This can help understand if the activity is related to legitimate behavior.
+
+- Investigation can be validated and reviewed based on the data that was matched and by viewing the source of that activity.
+- Consider the history of the vulnerability that was matched. Has it happened before? Is it happening on multiple machines? These kinds of questions can help understand if the activity is related to legitimate behavior.
+- Consider the user and their role within the company: is this something related to their job or work function?
+"""
+references = [
+    "https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-threatintel.html",
+    "https://docs.elastic.co/integrations/ti_rapid7_threat_command"]
+risk_score = 99
+rule_id = "3a657da0-1df2-11ef-a327-f661ea17fbcc"
+setup = """
+
+## Setup
+
+This rule needs threat intelligence indicators to work.
+Threat intelligence indicators can be collected using an [Elastic Agent integration](https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html#agent-ti-integration),
+the [Threat Intel module](https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html#ti-mod-integration),
+or a [custom integration](https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html#custom-ti-integration).
+
+More information can be found [here](https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html).
+
+## Max Signals
+
+This rule is configured to generate more **Max alerts per run** than the default 1000 alerts per run set for all rules. This is to ensure that it captures as many alerts as possible.
+
+**IMPORTANT:** The rule's **Max alerts per run** setting can be superseded by the `xpack.alerting.rules.run.alerts.max` Kibana config setting, which determines the maximum alerts generated by _any_ rule in the Kibana alerting framework. For example, if `xpack.alerting.rules.run.alerts.max` is set to 1000, this rule will still generate no more than 1000 alerts even if its own **Max alerts per run** is set higher.
+
+To make sure this rule can generate as many alerts as it's configured in its own **Max alerts per run** setting, increase the `xpack.alerting.rules.run.alerts.max` system setting accordingly.
+
+**NOTE:** Changing `xpack.alerting.rules.run.alerts.max` is not possible in Serverless projects.
+"""
+severity = "critical"
+tags = [
+    "OS: Windows",
+    "Data Source: Elastic Endgame",
+    "Data Source: Windows",
+    "Data Source: Network",
+    "Data Source: Rapid7 Threat Command",
+    "Rule Type: Indicator Match",
```

Comment:
Although the rule is related to TI integration, I think we are technically not matching the threat indicators, but the vulnerabilities. Maybe `Rule Type: Vulnerability Match` makes more sense if that tag allowed to exist.

Reply:
For `Rule Type` we attempt to match any actual rule type in the SIEM. If I understand correctly, this is a threat match type rule as indicated in the example you sent. https://github.com/elastic/integrations/blob/dfc057e4908d28fbe9f676ee4e3e0bdbe67cd35b/packages/ti_rapid7_threat_command/kibana/security_rule/eaecc8f0-6704-11ed-80b2-9bbc46f73b72.json#L80C17-L80C30

If that is the case, we can change it to `Rule Type: Threat Match` and `Use Case: Vulnerability` should still be searchable.